.github/workflows/Code-Quality-check.yml

```
@@ -4,32 +4,30 @@ on:
 permissions:
-  pull-requests: write
   contents: read
-  issues: write
-  checks: write
-  statuses: write
 
 jobs:
   build:
     runs-on: ubuntu-latest
     outputs:
       maven_cache_key: ${{ runner.os }}-maven-${{ steps.cache.outputs.cache-hit }}
```

actionlint (1.7.12)

[error] property "cache" is not defined in object type {}

(expression)